x-pack/plugins/reporting/server/export_types/common/pdf/worker.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+require('../../../../../../../src/setup_node_env');
+require('./worker.ts');
```

Comment:
Im curious when Kibana distributable is built if the `*.ts` file will be available? If you [download Kibana](https://www.elastic.co/downloads/kibana), it seems all `*.ts` files are stripped out.

Reply:
That's right, I tested this locally, and the only the worker.js file remains with the contents of worker.ts transpiled.